docs/template-resolution-for-dotnet-cli.md

```diff
@@ -156,58 +155,37 @@ Argument (name) matching: exact or partial short name or name, optional.
         groups that match the language will be shown (exact match)
     -   The default language is not considered
 
-    -   *Example:* `dotnet new --list --language F#`
-    -   *Example:* `dotnet new con --list --language F#`
+    -   *Example:* `dotnet new list --language F#`
+    -   *Example:* `dotnet new list con --language F#`
 
 -   Type
     -   If user specified the type via `--type` option, only the template groups
         that match the type will be shown (exact match)
 
-    -   *Example:* `dotnet new --list --type project`
-    -   *Example:* `dotnet new con --list --type project`
+    -   *Example:* `dotnet new list --type project`
+    -   *Example:* `dotnet new list con --type project`
 
 -   Baseline (hidden)
     -   If user specified the baseline via `--baseline` option, only the template
         groups that match the baseline will be shown (exact match)
 
-    -   *Example:* `dotnet new --list --baseline standard`
-    -   *Example:* `dotnet new con --list --baseline standard`
+    -   *Example:* `dotnet new list --baseline standard`
+    -   *Example:* `dotnet new list con --baseline standard`
 
 -   Author
     -   If user specified the author via `--author` option, only the template
         groups that match the author will be shown (exact or partial match)
 
-    -   *Example:* `dotnet new --list --author Microsoft`
-    -   *Example:* `dotnet new con --list --author soft`
+    -   *Example:* `dotnet new list --author Microsoft`
+    -   *Example:* `dotnet new list con --author soft`
 
 -   Tags
     -   If user specified the tag via `--tag` option, only the template groups
         that match the tag will be shown (exact match on single tag)
 
-    -   *Example:* `dotnet new --list --tag Common`
-    -   *Example:* `dotnet new con --list --tag Common`
-
-### Template parameter filters
```

Comment:
since it is a developer doc, what about keeping it and linking to https://github.com/dotnet/templating/issues/4061? or at least mentioning it in that issue so we don't forget.